test/lsp/incremental-lsp-changes/incremental-lsp-changes.rec

```diff
@@ -7,5 +7,3 @@ Read: {"jsonrpc":"2.0","method":"textDocument/didChange","params":{"textDocument
 Read: {"jsonrpc":"2.0","method":"textDocument/didChange","params":{"textDocument":{"uri":"file:///Users/dmitry/stripe/pay-server/api/lib/compatibility/change/foo.rb","version":3},"contentChanges":[{"range":{"start":{"line":3,"character":12},"end":{"line":3,"character":12}},"rangeLength":0,"text":"+"}]}}
 Write: {"jsonrpc":"2.0","method":"textDocument/publishDiagnostics","params":{"uri":"file:///Users/dmitry/stripe/pay-server/api/lib/compatibility/change/foo.rb","diagnostics":[{"range":{"start":{"line":4,"character":4},"end":{"line":4,"character":7}},"severity":1,"code":2001,"message":"unexpected token kEND","relatedInformation":[{"location":{"uri":"https://srb.help/2001","range":{"start":{"line":0,"character":0},"end":{"line":0,"character":0}}},"message":"Click for more information on this error."}]}]}}
 Read: {"jsonrpc":"2.0","method":"textDocument/didChange","params":{"textDocument":{"uri":"file:///Users/dmitry/stripe/pay-server/api/lib/compatibility/change/foo.rb","version":4},"contentChanges":[{"range":{"start":{"line":3,"character":13},"end":{"line":3,"character":13}},"rangeLength":0,"text":"\"\""}]}}
-Write: {"jsonrpc":"2.0","method":"textDocument/publishDiagnostics","params":{"uri":"file:///Users/dmitry/stripe/pay-server/api/lib/compatibility/change/foo.rb","diagnostics":[{"range":{"start":{"line":3,"character":8},"end":{"line":3,"character":15}},"severity":1,"code":7003,"message":"Method `Foo` does not exist on `Foo`","relatedInformation":[{"location":{"uri":"file:///Users/dmitry/stripe/pay-server/api/lib/compatibility/change/foo.rb","range":{"start":{"line":3,"character":8},"end":{"line":3,"character":11}}},"message":"Replace with `Foo.new`"},{"location":{"uri":"https://srb.help/7003","range":{"start":{"line":0,"character":0},"end":{"line":0,"character":0}}},"message":"Click for more information on this error."}]}]}}
```

Comment:
This seems accidental change. If you were to revert it, it would still work, right?